extensions/amp-form/amp-form.md

```diff
@@ -291,6 +296,38 @@ This shows all validation errors on all invalid inputs when the form is submitte
 #### As You Go
 This allows your user to see validation messages as they're interacting with the input, if the email they typed is invalid they'll see the error right away and once fixed the error goes away.
 
+## Variable Substitutions
+__(<a href="https://www.ampproject.org/docs/reference/experimental.html">experimental</a>)__
+
+`amp-form` allows [variable substitutions](../../spec/amp-var-substitutions.md) for inside hidden inputs with `default-value` attribute. On each submission, `amp-form` finds all `input[type=hidden]` inside the form and apply variable substitutions to `default-value` attribute and fill the result in the input `value`.
```

Comment:
Sent out #6621 to update the format we're doing here. PTAL there and I'll update docs PR after we get it in.